sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A code-searching tool similar to ack, but faster"
+DESCRIPTION="A code-searching tool similar to ack, but faster.
+* It searches code about 3–5× faster than ack.
+* It ignores file patterns from your .gitignore and .hgignore.
+* If there are files in your source repo you don't want to search, just add \
+their patterns to a .agignore file.
+* Ag uses Pthreads to take advantage of multiple CPU cores and search files in \
+parallel."
+HOMEPAGE="https://github.com/ggreer/the_silver_searcher/"
+COPYRIGHT="2013-2016 Geoff Greer"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="git+https://github.com/ggreer/the_silver_searcher.git#0.32.0"
+
+ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+    commandSuffix=
+    commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	the_silver_searcher$secondaryArchSuffix = $portVersion
+	cmd:ag$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:liblzma${secondaryArchSuffix}
+	lib:libpcre${secondaryArchSuffix}
+	lib:libz${secondaryArchSuffix}
+	"
+
+BUILD_REQUIRES="
+	devel:liblzma${secondaryArchSuffix}
+	devel:libpcre${secondaryArchSuffix}
+	devel:libz${secondaryArchSuffix}
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:make
+	cmd:gcc${secondaryArchSuffix}
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	aclocal
+	autoconf
+	autoheader
+	automake --add-missing
+
+	#Need to override the pkgconfig variables
+	LZMA_CFLAGS="-I$includeDir" \
+	LZMA_LIBS="-L$libDir -llzma" \
+	PCRE_CFLAGS="-I$includeDir" \
+	PCRE_LIBS="-L$libDir -lpcre" \
+	runConfigure --omit-dirs binDir ./configure \
+	--bindir=$commandBinDir
```

Comment:
Hmm, I think this is OK as it is.

Well, I'd just add an extra tab before **`--bindir=$commandBinDir`**
**_or_** join the last two lines into a single one:
```
	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
```